src/components/addBook.js

```diff
@@ -23,18 +22,19 @@ const AddBook = () => {
 
   const onClickInput = () => {
     dispatch(addBook({
-      id, title, author, completed: `${completed}%`, chapter: `${chapter}`,
+      id, title, author, category,
     }));
     setTitle('');
     setAuthor('');
+    setTimeout(() => dispatch(fetchBooks()), 500);
   };
 
   return (
     <form className="form">
       <h3 className="add-title">ADD NEW BOOK</h3>
       <div className="inputs">
-        <input className="add-book" type="text" placeholder="Book Title" onClick={onChangeHandlerTitle} />
-        <input type="text" placeholder="add-author" onChange={onChangeHandlerAuthor} value={author} />
+        <input className="add-book" type="text" placeholder="Book Title" onChange={onChangeHandlerTitle} value={title} />
+        <input type="text" placeholder="Author" className="add-author" onChange={onChangeHandlerAuthor} value={author} />
         <input className="btn-add-book" type="button" value="ADD BOOK" onClick={onClickInput} />
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** I think it would be better if you make all your inputs required to prevent the user to submit an empty form. You can add the `required` attribute in all your inputs. Right now, once clicked the **Add Book** button without filling in the inputs, it submits an empty form.